test/integration.sh

```diff
@@ -2,9 +2,6 @@
 
 # Runs a semi-realistic integration test with two servers, a stub authorization server, a 
 # prometheus that scrapes from them, and a single client that fetches "cluster" metrics.
```

Comment:
if required I can add the args to work with other ports and tokens but I think is not neccesary